charts/langsmith/docs/MIGRATE-OAUTH.md

```diff
@@ -0,0 +1,40 @@
+# Migrating from no-auth to OAuth
+
+We have provided a migration script to migrate your no-auth tenant to an oauth tenant. The script is located in the `scripts` directory of this repository.
+
+### Prerequisites
+
+Ensure you have the following tools/items ready.
```

Comment:
This should say the script needs to run after they add oauth and login to the UI with the desired "admin user" at least once

Reply:
fixed